server/src/main/java/org/elasticsearch/action/admin/indices/get/GetIndexResponse.java

```diff
@@ -249,15 +252,33 @@ public XContentBuilder toXContent(XContentBuilder builder, Params params) throws
                     }
                     builder.endObject();
 
-                    builder.startObject("mappings");
                     ImmutableOpenMap<String, MappingMetaData> indexMappings = mappings.get(index);
-                    if (indexMappings != null) {
+                    // the default on 6.x should be true to include types in the response
+                    boolean includeTypeName = params.paramAsBoolean(INCLUDE_TYPE_NAME_PARAMETER, true);
```

Comment:
I now have this default setting define in the constant BaseRestHandler.DEFAULT_INCLUDE_TYPE_NAME_POLICY

Reply:
Thanks